packages/core/src/time-field/time-field-hidden-input.tsx

```diff
@@ -0,0 +1,24 @@
+import type { ComponentProps } from "solid-js";
+
+import { useFormControlContext } from "../form-control";
+import { useTimeFieldContext } from "./time-field-context";
+
+export interface TimeFieldHiddenInputProps extends ComponentProps<"input"> {}
+
+export function TimeFieldHiddenInput(props: TimeFieldHiddenInputProps) {
+	const formControlContext = useFormControlContext();
+	const context = useTimeFieldContext();
+
+	return (
+		<input
+			type="text"
+			hidden
+			name={formControlContext.name()}
+			value={context.formattedValue() || ""}
+			required={formControlContext.isRequired()}
+			disabled={formControlContext.isDisabled()}
+			readOnly={formControlContext.isReadOnly()}
+			{...props}
+		/>
```

Comment:
`onChange` is needed here, while not used by users, browsers will recognize the field and autofill or cache the value when you refresh the form.

`onChange` should parse the text value and update `context.setValue(...)`.